code/modules/mob/living/simple_animal/friendly/sloth.dm

```diff
@@ -37,6 +39,15 @@
 /mob/living/simple_animal/sloth/Initialize(mapload)
 	. = ..()
 	AddElement(/datum/element/pet_bonus, "slowly smiles!")
+	// If someone adds non-cargo sloths to maps we'll have a problem but we're fine for now
```

Comment:
Why not move this to cargo’s sloths then?

Reply:
- citrus and paperwork don't have a shared parent besides `/sloth`
- i didn't wanna copy+paste the code
- i also didn't wanna repath all the sloths in this pr